CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Your contribution here.
 * [#1893](https://github.com/ruby-grape/grape/pull/1893): Allows `Grape::API` to behave like a Rack::app in some instances where it was misbehaving - [@myxoh](https://github.com/myxoh).
 * [#1898](https://github.com/ruby-grape/grape/pull/1898): Refactor ValidatorFactory to improve memory allocation - [@Bhacaz](https://github.com/Bhacaz).
+* [#1900](https://github.com/ruby-grape/grape/pull/1900): Define boolean for grape::api::instance - [@Bhacaz](https://github.com/Bhacaz).
```

Comment:
Correct the casing of `Grape::Api::Instance` and quote it. Also quote `ValidatorFactory` above while we're at it, please.